introduction/what-is.md

```diff
@@ -0,0 +1,45 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+The basic idea of InnerSource is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
+The producing team partners to review and accept the pull request and the functionality it implements.
+In this context of a code submission, we call the consuming team the **Guest** team and the producing team the **Host** team.
+
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
+In that situation, most people want to be a good host.
+They ensure that things are kept neat and tidy in anticipation of their guests' arrival.
+Visitors are greeted at the door and invited in.
+They can use the features and utilities that are in the home's public areas.
+There may be a few house rules that guests are asked to follow.
+Similarly, most guests want to show respect for the home and their host.
+They're careful with the items in the house and follow the rules for the duration of their stay.
+They may hope for or expect a return invitation as long as they've been courteous and polite.
+These concepts around a home visit are a metaphor for the attitude and behaviors that teams should bring as one hosts another making a guest contribution to the codebase.
+
+Let's take a closer look at how the mechanics of the inner source process can work.
+To aid in this explanation, we'll name a few key individuals on the guest and host teams.
+First, the **Product Owner** determines what functionality the host team is willing to accept as a contribution.
+The **Contributor** is the individual that submits the code contribution.
+The **Trusted Committer** provides any timely support and mentorship that the contributor needs to successfully submit the pull request.
+
+With those definitions, here is the basic outline of an inner source contribution.
+
+* Guest team requests a feature from the host team.
```

Comment:
This paragraph describes _one_, possibly the most common application of InnerSource. But it's not the only one. Should we also mention that sometimes people just contribute an improvement on their own initiative, like in Open Source? Perhaps we could lead into this paragraph by referring to the example in the last article.

Reply:
I'm not sure.  There isn't any reference here to motivations for _why_ the **Contributor** wants to get involved.  Doesn't what's written here apply equally to those contributing of their own free will just as much as those contributing as part of getting in a dependency for planned work?

Reply:
Made a small update to specify that a team or an contributor may be driving the ask for a change in source code.